docs/content/sidebars/concepts.js

```diff
@@ -6,7 +6,7 @@ const concepts = [
 	'concepts/components',
 	{
 		type: 'category',
-		label: 'Sui GraphQL RPC',
+		label: 'Sui GraphQL',
```

Comment:
I think I may have been responsible for putting this so high up in the "Concepts" sidebar, but I kind of regret that decision. Thoughts on moving it lower down, e.g. next to "Sui Architecture"?

Reply:
Ha. Moved into App Developers. I wouldn't be against SA, but just thought app devs would be more interested than someone looking into architecture. URL remains the same, so easy to move. Let me know.

Reply:
In this case, I meant to keep it as a top-level thing, but put it next to (just above?) "Sui Architecture" rather than nested within it, because there are reasons to use RPC outside of app development, but will leave it with you!